core/src/main/java/org/arquillian/smart/testing/configuration/ConfigurationItem.java

```diff
@@ -1,10 +1,13 @@
 package org.arquillian.smart.testing.configuration;
 
+import java.util.regex.Pattern;
+
 public class ConfigurationItem {
 
     private final String paramName;
     private String systemProperty;
     private Object defaultValue;
+    private Pattern pattern;
```

Comment:
btw, instead of using `Pattern` I would just expect that if system property ends with `.*` then it should go through all system properties and gets everything that starts with the string from it and store it as a List

Reply:
It is not necessary, it was used before but after code review I delete all code associated but I left erroneously the field